code/modules/clothing/shoes/magboots.dm

```diff
@@ -31,6 +31,9 @@
 	gustprotection = TRUE
 
 /obj/item/clothing/shoes/magboots/attack_self(mob/user, forced = FALSE)
+	toggle_magpulse(user, forced)
+
+/obj/item/clothing/shoes/magboots/proc/toggle_magpulse(mob/user, forced)
```

Comment:
The forced = FALSE argument should probably be moved to toggle_magpulse itself, so we dont have the arbitrary forced parameter on attack_self
```suggestion
/obj/item/clothing/shoes/magboots/attack_self(mob/user)
	toggle_magpulse(user)

/obj/item/clothing/shoes/magboots/proc/toggle_magpulse(mob/user, forced = FALSE)
```